### ISSUE TITLE
Accessibility - include alt tags in several places throughout site

### ISSUE THREAD

**Issue by kaylaondracek** [mlibrary/fishrappr#174]
## Description: 

Include alt tags where missing. These were discovered during Colin's accessibility review and during a previous screen reader review.

## Acceptance Criteria:

- [x] A. include alt tags on generated thumbnails for search results
- [x] B. include alt tags on "Today in the Michigan Daily" section
- [ ] ~~C. include alt tags in image-tiles that appear in viewer~~

**Comment by gordonleacock** [comment]
Code change for A. in application helper:

def hathitrust_thumbnail(document, **kw)
    namespace = document.fetch('ht_namespace')
    return fake_image(document, kw.fetch(:size, ',250')) if namespace == 'fake'
    %Q{<img src="#{hathitrust_thumbnail_src(document, **kw)}" alt="Thumbnail of Daily page" />}.html_safe
  end

**Comment by gordonleacock** [comment]
@roger: Can you give me suggestions on how to do ac C.?

**Comment by respinos** [comment]
That's no longer relevant. 

R
